librz/core/rtr_http.c

```diff
@@ -486,7 +483,7 @@ the_end : {
 	const char *port = rz_config_get(core->config, "http.port");
 	const char *cors = rz_config_get(core->config, "http.cors");
 	const char *allow = rz_config_get(core->config, "http.allow");
-	core->config = origcfg;
+
```

Comment:
Probably you want to remove all the `rz_config_get` and `rz_config_set` above and below, because these calls makes no sense.